lib/albacore/task_types/find_msbuild_versions.rb

```diff
@@ -20,6 +20,11 @@ def self.find_msbuild_versions
     rescue
       error "failed to open HKLM\\SOFTWARE\\Microsoft\\MSBuild\\ToolsVersions"
     end
+    
+    # MSBuild 15, assume default installation path
+    vs2017_dir = Dir[File.join(ENV['ProgramFiles(x86)'].gsub('\\', '/'), 'Microsoft Visual Studio', '2017', '*')].first
+    retval[15] = File.join(vs2017_dir, 'MSBuild', '15.0', 'Bin') unless vs2017_dir.nil?
+    
```

Comment:
Okay I'm even sillier than I thought. This is the old version before my change. Sigh. :-P Please ignore me today.